src/thx/bigint/Bigs.hx

```diff
@@ -383,8 +383,17 @@ class Bigs {
 			b_l = b.length,
 			result = createFloatArray(b.length),
 			divisorMostSignificantDigit:Float = b[b_l - 1]#if (neko || eval) + 0.0 #end, // normalization
-		lambda = Math.ceil(BASE / (2 * divisorMostSignificantDigit)), remainder:Array<Float> = multiplySmall(a,
-			lambda).map(function(v):Float return v), divisor = multiplySmall(b, lambda), quotientDigit:Float, shift, carry:Float, borrow:Float, i, l, q:Float;
+		lambda = Math.ceil(BASE / (2 * divisorMostSignificantDigit)),
+		remainder:Array<Float>
+		= multiplySmall(a, lambda).map(function(v):Float return v),
```

Comment:
Very wonky formatting. Avoid by using vars instead of long expressions on the same line